### PR TITLE
chore: release 1.1.0

### PR DESCRIPTION
**Pull request by kilchenmann**
:robot: I have created a release \*beep\* \*boop\*
---
### Features

* always show project selection (DSP-1274) ([#1](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/1)) ([3174aec](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/3174aecbf0cf7a223fd3f8b5fd6ff1671e8238cf))
* **clipurl plugin:** add plugin (DSP-1494) ([#17](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/17)) ([352e99a](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/352e99ae50cbe1d5b14d52925b335474f63cb06e))
* **ext. search:** use labels for ext. search (DSP-1547) ([#18](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/18)) ([a8966d0](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/a8966d09bfc80153e42947466dd69bd83f8ed600))
* **extended search:** add case for URI value ([#15](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/15)) ([56fb28f](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/56fb28f5def2ec3067da838e04bea70d51a30c04))

### Bug Fixes

* **clickable links:** wrap URLs with anchor tags (DSP-1490) ([#13](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/13)) ([1cc8be8](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/1cc8be8a897bc9d002570a610e1f355ca6c0436f))
* **datepicker:** make day selection wider (DSP-1524) ([#16](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/16)) ([dd566b8](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/dd566b8b7949d273e93cdbb5f4b18e93b0b8d6b9))
* **ext search:** add case for Boolean value ([#12](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/12)) ([d3e69b8](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/d3e69b8b1a374a775777385d34c3ac37aa52079e))
* **geoname:** add missing logic for Geoname values (DSP-1373) ([#7](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/7)) ([36a7be6](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/36a7be60653820c5ccef3abb88f6605081cff3a3))
* **res creation:** prevent error when geoname value is empty ([#14](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/14)) ([85dc51b](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/85dc51b2f13a44a051ffd618b083819999411f6d))
* **window controls:** keep ctrl buttons visible (DSP-1384) ([#8](https://www.github.com/dasch-swiss/dsp-tangoh-app/issues/8)) ([7613425](https://www.github.com/dasch-swiss/dsp-tangoh-app/commit/7613425e521d92e31e7adbeca78c963c130de2c3))
---

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-13.0.0
+1.1.0
```
Comment (tobiasschweizer) [inline version.txt:1]: Are these a remains of DSP-API's version?

Comment (kilchenmann) [reply version.txt:1]: I think the one who added release-please copied from DSP-API repo but didn't adjust the version number 😉

Comment (tobiasschweizer) [reply version.txt:1]: ok, got it

**Review by tobiasschweizer — APPROVED** [review]
let's try

**Comment by kilchenmann** [comment]
:robot: Release is at https://github.com/dasch-swiss/dsp-tangoh-app/releases/tag/v1.1.0 :sunflower:

**Pull request merged**